metadata-db-app/src/cmr/metadata_db/data/oracle/sql_helper.clj

```diff
@@ -53,6 +65,14 @@
                  (dissoc params :concept-type)
                  (dissoc params :concept-type :provider-id))
         table (ct/get-table-name provider concept-type)
+        get-stmt (su/build (select [:concept-id :revision-id]
+                                   (from table)
+                                   (where (find-params->sql-clause params))))
+        get-values (when (not (= "efs-off" (efs-config/efs-toggle)))
+                     (j/query db get-stmt))
         stmt (su/build (delete table
-                         (where (find-params->sql-clause params))))]
+                               (where (find-params->sql-clause params))))]
+    (when (not (= "efs-off" (efs-config/efs-toggle)))
```

Comment:
nit; maybe this could be `(when = "efs-on" (efs-config/efs-toggle))` but, as long as it is consistent throughout the other toggles that is more important

Reply:
It's like this because there are two options for efs begin on, 'efs-on' and 'efs-only'. Originally I checked for multiple values, but this not check was more concise without really changing function